Apps/phironnet/ironnet_connector.py

```diff
@@ -345,6 +352,67 @@ def _handle_irondefense_get_alert_irondome_info(self, param):
             self.debug_print("Retrieving IronDome alert info failed. Error: {}".format(action_result.get_message()))
             return action_result.set_status(phantom.APP_ERROR, "Retrieving IronDome alert info failed. Error: {}".format(action_result.get_message()))
 
+    def _handle_irondefense_get_alert_notifications(self):
+            self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+
+            # Add an action result object to self (BaseConnector) to represent the action for this param
+            action_result = self.add_action_result(ActionResult(dict()))
+
+            request = {
+                'limit': self._alert_limit
+            }
+
+            # make rest call
+            ret_val, response = self._make_post('/GetAlertNotifications', action_result, data=request, headers=None)
+            if phantom.is_success(ret_val):
+                self.save_progress("Fetching alert notifications was successful")
+                # Filter the response and add into the data section
+                for alert_notification in response['alert_notifications']:
+                    if alert_notification['alert_action'] in self._alert_notif_actions and alert_notification['alert']:
+                        alert = alert_notification['alert']
+                        if alert['category'] not in self._alert_categories and alert['sub_category'] not in self._alert_subcategories:
+                            if self._alert_severity_lower <= int(alert['severity']) <= self._alert_severity_upper:
+                                # create container
+                                contnr = {
```

Comment:
Not sure why - but `contnr` really bugs me.  Can we rename to `container` please? :)